ui/widgets/nuxeo-user-avatar.js

```diff
@@ -299,33 +307,40 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
           let tempName = '';
           const splitName = name.split(' ');
```

Comment:
nit: This is only used for the else statement, correct? I would suggest to only have it there.